gold-phone-input.html

```diff
@@ -78,7 +85,7 @@
         always-float-label="[[_computeAlwaysFloatLabel(alwaysFloatLabel,placeholder)]]"
         invalid="[[invalid]]">
 
-      <label style="left:40px;" hidden$="[[!label]]">[[label]]</label>
+      <label hidden$="[[!label]]">[[label]]</label>
```

Comment:
Turns out this gets it by magic, but that should probably be fixed too: `gold-phone-input` has a `paper-input-container` which imports all of `paper-styles`, and with it `iron-flex-layout/classes` which has `[hidden]`

:boom: